app/views/jobs/new.html.erb

```diff
@@ -0,0 +1,88 @@
+<% provide :title,  t(".job")  %>
+<center><h1 class="text-uppercase mt-3"><%= t ".job" %></h1></center>
+
+<div class="row justify-content-center mt-5">
+  <div class="col-md-8">
+
+    <%= form_with model: @job do |f| %>
+      <%= render "shared/error_messages", object: f.object %>
+
+      <div class="form-group">
+        <%= f.label :organization_id, t(".organization_id") %><br>
+        <%= f.select :organization_id, options_for_select(@organization_options),
+          {include_blank: t(".choose_organization")}, {class: "form-control"} %>
+      </div>
+      
+      <div class="form-group">
+        <%= f.label :title, t(".title") %>
```

Comment:
dùng i18n cho activerecord nhé em
https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models